webui/src/pages/repositories/wizard/spark_config_step.jsx

```diff
@@ -24,14 +24,27 @@ async function uploadAndCommitReadme(repoId, branchName, importLocation) {
     await commits.commit(repoId, branchName, 'Add Spark quickstart README', {user: 'Spark quickstart'},);
 }
 
+async function uploadAndCommitSampleNotebook(repoId, branchName, importLocation) {
+    const SPARK_SAMPLE_NOTEBOOK_TEMPLATE_NAME = 'python.notebook.ipynb.tt';
+    const notebookProp = { repo: repoId, branch: branchName, ...lakeFSURLProp };
+    if (importLocation) {
+        notebookProp['import_location'] = importLocation;
+    }
+    const notebookConfig = await templates.expandTemplate(SPARK_SAMPLE_NOTEBOOK_TEMPLATE_NAME, notebookProp);
+    const notebookFile = new File([notebookConfig], 'Spark Demo.ipynb', {type: 'application/x-ipynb+json'});
```

Comment:
Personally, I don't like whitespace and caps in filename.

Reply:
Would mention this notebook in the README file as well, together with a link

Reply:
I agree. Removed whitespaces and made all lower case.